examples/code/example_common.h

```diff
@@ -58,9 +58,9 @@ namespace
         main_scene = put::ecs::create_scene("main_scene");
 
         put::camera_create_perspective(&main_camera, 60.0f, put::k_use_window_aspect, 0.1f, 1000.0f);
-
+        ImGuiStyle &style = ImGui::GetStyle();
```

Comment:
`put::dev_ui::init();` can handle this with the default argument, I expect to see no changes to this file in this pull request.